docs/cookbook/setting_up_multi_language_siteaccesses.md

```diff
@@ -144,3 +144,9 @@ php bin/console cache:clear
 ```
 
 You should now be able to load your eZ Platform site in the `en` and `fr` SiteAccess displaying English and French content.
+
+!!! tip
+    
+    If you encounter issues when configuring SiteAccess, use the following command to debug or speed up the configuration process:
```

Comment:
In what way do we imply this will speedup the process? What did you have in mind here? :)

Reply:
I must have misunderstood something 😅 I will remove that fragment 😄

Reply:
Could be something like:
`If you encounter issues when configuring SiteAccess or want to check that system picks up the right value, use the following command:`